src/db/database.provider.ts

```diff
@@ -23,7 +24,7 @@ export const databaseProviders = [
           config = databaseConfig.development;
       }
       const sequelize = new Sequelize(config);
-      sequelize.addModels([User,Post]);
+      sequelize.addModels([User,Post,Comment]);
```

Comment:
Instead of importing individual models, we can import and export into a file and import the file here